frontend/src/routes/SearchPage/components/HeaderWithNotification.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2021 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+import React from 'react'
+import '@patternfly/react-core/dist/styles/base.css'
+import { useTranslation } from 'react-i18next'
+import { AcmPageHeader, AcmInlineStatus, StatusType } from '@open-cluster-management/ui-components'
+import { Card, CardBody } from '@patternfly/react-core'
+
+// 1. Needed to export as default function.
+export default function HeaderWithNotification(props: { queryMessages: any[] }) {
+    const { t } = useTranslation(['search'])
+    const messages: any[] = props.queryMessages
+
+    return (
+        <div>
+            <div style={{ outline: 'none', display: 'flex', justifyContent: 'flex-end' }}>
+                <p style={{ flex: 1 }}>
+                    <AcmPageHeader title={t('search')} />
+                </p>
```

Comment:
Change `<p>` to `<div>` here to resolve these problems in the unit test.
![image](https://user-images.githubusercontent.com/4671325/122955792-6b894480-d34e-11eb-8f76-e6696f324224.png)
```suggestion
                <div style={{ flex: 1 }}>
                    <AcmPageHeader title={t('search')} />
                </div>
```